x/margin/keeper/query_is_whitelisted.go

```diff
@@ -9,15 +9,16 @@ import (
 	"google.golang.org/grpc/status"
 )
 
-func (k Keeper) IsWhitelisted(goCtx context.Context, req *types.IsWhitelistedRequest) (*types.IsWhitelistedResponse, error) {
+func (k Keeper) GetIsWhitelisted(goCtx context.Context, req *types.IsWhitelistedRequest) (*types.IsWhitelistedResponse, error) {
 	if req == nil {
 		return nil, status.Error(codes.InvalidArgument, "invalid request")
 	}
 
 	ctx := sdk.UnwrapSDKContext(goCtx)
+	IsWhitelisted := k.IsWhitelisted(ctx, req.Address)
```

Comment:
Would be better to use isWhitelisted than IsWhitelisted

Reply:
@jelysn I got it.

Reply:
@jelysn Done, removed.